packages/core/useWheelScrollDirection/index.ts

```diff
@@ -1,6 +1,8 @@
-import { unref } from 'vue-demi'
+import { computed, unref } from 'vue-demi'
 import type { MaybeRef } from '@vueuse/shared'
+import type { MaybeElementRef } from '@vueuse/core'
 import { useEventListener } from '@vueuse/core'
+import { unrefElement } from '../unrefElement'
 
```

Comment:
Maybe it can be written like this `import { unrefElement, type MaybeElementRef } from '../unrefElement'`

Reply:
Yes, but I think it would be better to separate.

Reply:
ok.